### ISSUE TITLE
h1 of note isn't dynamic

### ISSUE THREAD

**Issue by konhi** [obsidian-community/obsidian-hub#7]
# Actual Behavior
1. before changing the title
![image](https://user-images.githubusercontent.com/61631665/134785076-3a24db40-f837-4503-8d88-98bd3ed87efa.png)
2. after changing the title
![image](https://user-images.githubusercontent.com/61631665/134785115-6554b9c0-5e7f-409f-b7cd-3ef179cc9c08.png)

# Suggested Behavior
1. before changing the title
![image](https://user-images.githubusercontent.com/61631665/134785092-77e71930-d4ed-4d41-add9-0eb68051c1b9.png)
2. after changing the title
![image](https://user-images.githubusercontent.com/61631665/134785140-9739e40c-0de0-415b-96b2-eb590fa9a7bc.png)

---

- [ ] **Let me know if it should be changed to the suggested behavior.**

**Comment by tallguyjenks** [comment]
This is why the titles for notes in my vault are all links to themselves, so they auto update when the note title changes :) 👍🏻 

```md
[[Note Title]]
```

**Comment by konhi** [comment]
> This is why the titles for notes in my vault are all links to themselves, so they auto update when the note title changes :) 👍🏻 
> 
> ```md
> [[Note Title]]
> ```

Yes, exactly! I learnt that trick from your awesome YouTube videos! If you think that there could be other enhancements like that for this vault, please let us know!

**Comment by kometenstaub** [comment]
We could also remove all h1 headings and remove the css rule for not displaying the title, I think that would be the easiest way.

**Comment by konhi** [comment]
👍 I agree! I personally stopped using title as h1 in my Vault. Instead I use [obsidian-embedded-note-titles](https://github.com/mgmeyers/obsidian-embedded-note-titles).

**Comment by argenos** [comment]
Just for context: I added H1 titles in case we wanted to deviate from/remove the JD numbering and/or we had shorter file names and a more descriptive title. But I'm also ok with removing it

**Comment by konhi** [comment]
> 
> 
> Just for context: I added H1 titles in case we wanted to deviate from/remove the JD numbering and/or we had shorter file names and a more descriptive title. But I'm also ok with removing it

I see! Then is it decided, can we do a PR?

**Comment by argenos** [comment]
Hmmm I'm doing some edits now, and I think removing them won't be an option, e.g. community plugin file names are their ID (`nldates-obsidian`) and the H1 is the name of the plugin (which looks a bit better but on occasion contains forbidden characters). Is there some CSS trickery to ignore the file name for notes with H1? Then we can have a mix when it makes sense

**Comment by argenos** [comment]
Not super happy with how that looks, but I guess that could also be modified with CSS, so yeah! Are there many notes where H1 and file name don't match? Do we expect a lot of file renaming?

**Comment by konhi** [comment]
> Are there many notes where H1 and file name don't match? Do we expect a lot of file renaming?

I actually didn't see any occurencies of that! It's not really that important change.
